vizro-ai/docs/pages/explanation/safety-in-vizro-ai.md

```diff
@@ -62,4 +61,4 @@ Users must exercise caution when executing code generated by or influenced by AI
 - Always be aware that malicious code execution cannot be mitigated with 100% certainty
 - Always review and understand the selected model before connecting with `vizro_ai`
 
-If you would like to learn more about our efforts in safeguarding code execution, please refer to [Safeguard Execution of Dynamic Code](safeguard.md) for more information.
+If you would like to learn more about our efforts in safeguarding code execution, refer to [Safeguard Execution of Dynamic Code](safeguard.md) for more information.
```

Comment:
I am always confused what we capitalise and what we keep lower-case e.g. below I would have replicated how the title of that section is actually displayed so only capitalising the first letter? 🤔 

```suggestion
If you would like to learn more about our efforts in safeguarding code execution, refer to [Safeguard execution of dynamic code](safeguard.md) for more information.
```

Reply:
You're right, but in fact that was all a bit of a mess, so I've fixed it in your suggested edit. Thanks! 🎯